src/ngraph/runtime/gpu/gpu_util.cpp

```diff
@@ -32,14 +32,15 @@ using namespace std;
 
 void runtime::gpu::print_gpu_f32_tensor(void* p, size_t element_count, size_t element_size)
 {
-    float* local;
+    float* local = new float[element_count];
```

Comment:
nitpick: we have a helper `vector_to_string` I'd just recommend allocating the vector of the right length getting `&v[0]` and using `vector_to_string` to print out tensors' values.

Reply:
Or use the `join` function in util.hpp